.github/workflows/wasm-benchmarks.yml

```diff
@@ -59,6 +59,7 @@ jobs:
 
       - name: Run benchmarks
         id: bench
+        if: false
```

Comment:
Note for posteriority: temprarily disabled, as we can't run benchmarks against older version of WASM engine without using driver adapters too: https://github.com/prisma/team-orm/issues/1152